### ISSUE TITLE
Exploded view

### ISSUE THREAD

**Issue by jaredjennings** [jaredjennings/dactyl-keyboard#22]
I've taken great care to make sure all the pieces are in the right places to fit together properly. Now, to show people the pieces, I want to explode them apart.

The way I would explain it is that each piece should be moved by a certain amount along the vector between the centroid of all pieces and the centroid of the piece. http://forum.openscad.org/Centre-of-Mass-td14707.html seems to suggest that you can't do this sort of thing in OpenSCAD, so it will likely necessitate using another program. But it should be automated so I don't have to worry about re-doing it manually all the time.

**Comment by jaredjennings** [comment]
Well - Blender can do it in one step, https://blender.stackexchange.com/a/68465.

**Comment by jaredjennings** [comment]
At great length, I took that step in [f567a9db]. The thing I feared happened: it was easy to move the pieces apart in the way described; and then it was easy to start tweaking colors, lights and positions; and then I did a bunch of tweaking. The work of importing the STLs is easy enough that dealing with changes in the shapes of the pieces will not be too onerous, although the data is duplicated in the blend files. Not as good as a blend file with a python script in it that repeatably does the same thing, nor nearly so small, but much quicker.
